cogs/custom.py

```diff
@@ -38,14 +44,58 @@ async def vote(self, ctx: Context, title: str, *options: str) -> None:
             raise BadArgument("Please provide at least 2 options.")
         if len(options) > 20:
             raise BadArgument("I can only handle 20 options!")
-
         codepoint_start = 127462  # represents "regional_indicator_a" unicode value
         options = {chr(i): f"{chr(i)} - {v}" for i, v in enumerate(options, start=codepoint_start)}
         embed = Embed(title=title, description="\n".join(options.values()))
         message = await ctx.send(embed=embed)
         for reaction in options:
             await message.add_reaction(reaction)
 
+    @command()
+    async def members(self, ctx: Context) -> None:
+        """Returns the number of members in a server."""
+        await ctx.send(embed=Embed(title="Member count", description=ctx.guild.member_count, color=Color.dark_purple()))
+
+    @command()
+    async def status(self, ctx: Context, member: t.Optional[Member] = None) -> None:
+        """Returns the status of a member."""
+        if member is None:
+            member = ctx.author
+
+        if member.id == constants.owner_ids:
```

Comment:
`constants.owner_ids` is a list, `member.id` is an integer, they will never be the same, this should be:
```suggestion
        if member.id in constants.owner_ids:
```